performance-tracking/src/main/java/com/rakuten/tech/mobile/perf/runtime/internal/AnalyticsBroadcaster.java

```diff
@@ -0,0 +1,72 @@
+package com.rakuten.tech.mobile.perf.runtime.internal;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.annotation.VisibleForTesting;
+import android.support.v4.content.LocalBroadcastManager;
+import android.text.TextUtils;
+import com.rakuten.tech.mobile.perf.core.Analytics;
+import java.lang.ref.WeakReference;
+import java.util.HashMap;
+import java.util.Map;
+
+/** Broadcaster that sends events via local broadcast to peer analytics sdk. */
+class AnalyticsBroadcaster extends Analytics {
+  @VisibleForTesting
+  static final String ACTION = "jp.co.rakuten.sdtd.analytics.ExternalEvent";
```

Comment:
It seems really weird to use `jp.co.rakuten.sdtd.analytics.ExternalEvent`.

Should it be something like `com.rakuten.tech.mobile.perf.analytics.ExternalEvent` instead? But that would mean the analytics receiver needs to be updated...

Reply:
it is, but it makes integration easier. 
also a new action would require listening to more actions an the analytics side.